### ISSUE TITLE
Open detail pages in drawer when interacting on a table for faster navigation

### ISSUE THREAD

**Issue by wolflu05** [inventree/InvenTree#7364]
That issue leads me to another idea, what about opening the parts detail page, but only in a drawer with about 90% width so that one can easily step through parts, check things and quickly continue with the next part. (We could do that for every tables detail page). But if we open the url directly (so we're not coming from a table), then we open the part in 100% without loading and showing the table first

_Originally posted by @wolflu05 in https://github.com/inventree/InvenTree/issues/7271#issuecomment-2120497054_

**Comment by wolflu05** [comment]
I still like to see this implemented in some time, maybe also only when pressing a specific modifier key when clicking or behind a setting.

**Comment by wolflu05** [comment]
I still want to implement this at some time.

**Comment by matmair** [comment]
Was this implemented to the level you envisioned @wolflu05 ?

**Comment by wolflu05** [comment]
No, this is not implemented as far as I know. This is issue is about opening a drawer (~90% width) with the part page instead of navigating to the part page, like described in the initial issue description. This will make searching through the parts a lot easier, as the state with the table is kept in the background, and there is no need to open all potentially fitting parts in new browser tabs and then going 1 by 1 through them.

**Comment by matmair** [comment]
I have been thinking about this - I think having something like this in a generic way would be great to enable quick browsing etc

This could be either done as a secondary interaction for powerusers or as the primary one that has a prominent action to open a full page.

Having someone that does foundational UX design would really help sometimes

**Comment by SchrodingersGat** [comment]
I also agree that this would be a powerful improvement to the workflow.

- Click on item in table
- Brings up drawer focus view
- User can choose to click through to the full detail page if required

**Comment by matmair** [comment]
I think it would be valuable to have these patterns generalized and broadly used in the whole code base to enable a similar look and feel between core and plugins
